changelog.md

```diff
@@ -1,5 +1,9 @@
 # [5.0.0](https://github.com/TehShrike/deepmerge/releases/tag/v5.0.0)
 
+- Breaking: Endpoint are now exported in esm style [#215](https://github.com/TehShrike/deepmerge/pull/215)
```

Comment:
It's a bit tough for folks to go check out the PR to see exactly what the change was and run into
<img width="156" alt="image" src="https://user-images.githubusercontent.com/1141869/100524084-ab9e9500-317a-11eb-8574-a6e360f87d30.png">